src/Data/Swagger/Internal/Schema.hs

```diff
@@ -662,7 +665,10 @@ gdeclareNamedSumSchema opts proxy s
 
     toStringTag schema = mempty
       & type_ .~ SwaggerString
-      & enum_ ?~ map toJSON (schema ^.. properties.ifolded.asIndex)
+      & enum_ ?~ map toJSON'  (schema ^.. properties.ifolded.asIndex)
+ 
+    -- TODO:
```

Comment:
To remember to remove `toJSON'` - had to add explicit type it to get better error message. It's done now.